OpenAFSLibrary/keywords/path.py

```diff
@@ -236,4 +236,16 @@ def get_fid(self, path):
 
     def get_name_by_fid(self, fid):
```

Comment:
Suggest we change this to

    def get_name_by_fid(self, fid, server=None):

Then change the robot file like so:

    -|  | ${FID}=                    | Get FID          | ${PATH}/${FILE}
    +|  | ${FID}=                    | Get FID          | ${PATH}/${FILE}  | server=${SERVER}

Reply:
I think the changes should be: :) 

```

-|  | ${NAME}                    | Get Name by FID  | ${FID}
+|  | ${NAME}                    | Get Name by FID  | ${FID} | server=${SERVER}

```